### PR TITLE
Can't pry tiles with a crowbar while using harm intent

### PR DESCRIPTION
**Pull request by ghost**
Credit to https://github.com/d3athrow/vgstation13/pull/12681 for good idea.

### CODE REVIEW

**Comment by ghost** [comment]
oh god it shows that I referenced it on the /vg/ PR that's scary

File: code/game/turfs/simulated/floor.dm [hunk 1]
```diff
@@ -147,11 +147,15 @@ var/list/icons_to_ignore_at_floor_init = list("damaged1","damaged2","damaged3","
...
+			if (user.a_intent != "harm")
+				if(istype(src, /turf/open/floor/wood))
+					user << "<span class='danger'>You forcefully pry off the planks, destroying them in the process.</span>"
+				else
+					user << "<span class='danger'>You remove the floor tile.</span>"
+					builtin_tile.loc = src
 			else
-				user << "<span class='danger'>You remove the floor tile.</span>"
-				builtin_tile.loc = src
+				user << "<span class='notice'>You swing the crowbar in front of you.</span>"
```
Comment (TalkingCactus) [inline code/game/turfs/simulated/floor.dm]: Don't need this if nothing happened.

**Comment by ghost** [comment]
removed that for you, should be fine now

File: code/game/turfs/simulated/floor.dm [hunk 2]
```diff
@@ -147,11 +147,14 @@ var/list/icons_to_ignore_at_floor_init = list("damaged1","damaged2","damaged3","
 			burnt = 0
 			user << "<span class='danger'>You remove the broken plating.</span>"
 		else
-			if(istype(src, /turf/open/floor/wood))
-				user << "<span class='danger'>You forcefully pry off the planks, destroying them in the process.</span>"
+			if (user.a_intent != "harm")
+				if(istype(src, /turf/open/floor/wood))
+					user << "<span class='danger'>You forcefully pry off the planks, destroying them in the process.</span>"
+				else
+					user << "<span class='danger'>You remove the floor tile.</span>"
```
Comment (TalkingCactus) [inline code/game/turfs/simulated/floor.dm]: `You remove [src.name].`

File: code/game/turfs/simulated/floor.dm [hunk 3]
```diff
@@ -147,11 +147,14 @@ var/list/icons_to_ignore_at_floor_init = list("damaged1","damaged2","damaged3","
 			burnt = 0
 			user << "<span class='danger'>You remove the broken plating.</span>"
 		else
-			if(istype(src, /turf/open/floor/wood))
-				user << "<span class='danger'>You forcefully pry off the planks, destroying them in the process.</span>"
+			if (user.a_intent != "harm")
+				if(istype(src, /turf/open/floor/wood))
+					user << "<span class='danger'>You forcefully pry off the planks, destroying them in the process.</span>"
```
Comment (TalkingCactus) [inline code/game/turfs/simulated/floor.dm]: `You forcefully pry off [src.name], destroying \them in the process.`
note: never tried `\them`, so test it out to see if it works. not sure where the `\X` things are anyway

**Review by TalkingCactus — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
done

**Pull request merged**